src/TodoBtn.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import SampleModal from './SampleModal'
+
+interface TodoBtnProps {
+  saveData(id?: number): void
+  btnType: string
```

Comment:
타입을 string 에서 저 세가지 문구만 해당되도록 타입을 좁힐 수 있어요.

```suggestion
  btnType: '추가' | '수정' | '삭제'
```